### PR TITLE
Add accessibility recommendations 🌐

### PR DESCRIPTION
**Pull request by SiAdcock**
## What does this change?

In order to improve the accessibility of our products consistently, and to ensure they continue to be accessible in the future, this change adds a few accessibility recommendations.

There might be gaps around accessibility on native apps, so I'd appreciate some advice from native apps teams on this subject. 

I'm also happy to hear suggestions for any other high level recommendations - how can we cover 80% of accessibility issues using these recommendations?

### CODE REVIEW

**Review by rhystmills — APPROVED** [review]
This looks like a really useful document - I actually think the tone is right: efficient and straight to the point.

File: accessibility.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+Aim to build user interfaces that:
+
+- Provide [text alternatives](https://www.w3.org/TR/WCAG21/#text-alternatives)
+- Are [adaptable](https://www.w3.org/TR/WCAG21/#adaptable)
+- Are [distinguishable](https://www.w3.org/TR/WCAG21/#distinguishable)
+- Do not [trigger seizures or physical reactions](https://www.w3.org/TR/WCAG21/#seizures-and-physical-reactions)
+- Are [navigable](https://www.w3.org/TR/WCAG21/#navigable)
+- Are [predictable](https://www.w3.org/TR/WCAG21/#navigable)
+- Help users [avoid or correct mistakes](https://www.w3.org/TR/WCAG21/#input-assistance)
+- Are [compatible](https://www.w3.org/TR/WCAG21/#compatible) with existing or future users agents and assistive technologies
```
Comment (mxdvl) [inline accessibility.md:16]: I am a big fan of the [reference-style link notation][md] in Markdown. It keeps a better flow for longer sentences.

[md]: https://daringfireball.net/projects/markdown/syntax#link "I’m using it here, too!"

```suggestion
- Are [compatible][] with existing or future users agents and assistive technologies

[compatible]: (https://www.w3.org/TR/WCAG21/#compatible)
```

**Review by mxdvl — APPROVED** [review]
Important work 👏

File: accessibility.md (hunk 1 above)
Comment (SiAdcock) [reply accessibility.md:16]: Cool! Will the output look any different?

Comment (mxdvl) [reply accessibility.md:16]: Not at all!

File: accessibility.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+- Talkback on Android
+
+For all products, test changes using a screen magnifier:
+
+- [Magnifier on iOS](https://support.apple.com/en-gb/HT209517)
+- [Magnifier on Android](https://support.google.com/accessibility/android/answer/6006949?hl=en-GB)
+- [Zoom on OSX](https://support.apple.com/en-gb/guide/mac-help/mh40579/10.15/mac/10.15)
+- [Magnifier on Windows](https://support.microsoft.com/en-gb/help/11542/windows-use-magnifier-to-make-things-easier-to-see)
+
+Changes to web-based products should also be tested using only a keyboard.
```
Comment (SiAdcock) [inline accessibility.md:49]: Oops repetition

**Comment by jennygrahamjones** [comment]
Do we do / encourage any testing with speech recognition software, like Dragon?

**Comment by SiAdcock** [comment]
@jennygrahamjones I would love to provide recommendations around it, but I'm not personally familiar with this kind of software and I'm not sure if anyone else is (please speak up if I'm wrong 😆) 

If you have any experience with Dragon Naturally Speaking or anyhing similar, would you be up for a chat about this? It would be very useful if we could give people a starting point for how to go about testing with speech recognition software.

**Comment by jennygrahamjones** [comment]
> If you have any experience with Dragon Naturally Speaking or anything similar, would you be up for a chat about this? It would be very useful if we could give people a starting point for how to go about testing with speech recognition software.

@SiAdcock I don't have any personal experience testing with Dragon, unfortunately. It's something that I've only ever seen tested by users of Dragon themselves. Perhaps this type of testing falls into something to consider when organising external audits.

**Comment by SiAdcock** [comment]
@jennygrahamjones That is a good shout. Elsewhere, we could put together examples of what kind of things an external audit ought to be testing. An exhaustive list of assistive technologies (including different navigation hardware such as swicthes and eye trackers) would be useful.

**Review by jennygrahamjones — APPROVED** [review]

**Pull request merged**